src/Explorer/Controls/NotebookGallery/GalleryViewerComponent.tsx

```diff
@@ -69,6 +71,8 @@ interface GalleryViewerComponentState {
   searchText: string;
   dialogProps: DialogProps;
   isCodeOfConductAccepted: boolean;
+  isFetchingPublishedNotebooks: boolean;
+  isFetchingFavouriteNotebooks: boolean;
```

Comment:
Do we also need flags to track fetching of samples and public gallery?

Reply:
that is taken care of by the spinner in createCardsTabContent. These 2 cases were different since we showed a placeholder message if the fetched data is empty